src/transactions/PaymentOpFrame.cpp

```diff
@@ -68,8 +68,7 @@ PaymentOpFrame::doApply(Application& app, LedgerState& ls)
     opRes.code(opINNER);
     opRes.tr().type(PATH_PAYMENT);
     PathPaymentOpFrame ppayment(op, opRes, mParentTx);
-    // TODO(jonjove): This needs to be changed
-    ppayment.setSourceAccountPtr(mSourceAccount);
+    // TODO(jonjove): Used to set source account pointer here, why?
```

Comment:
that was because it's a different object from what I remember - not relevant anymore :)